config-connector/tests/ccs-test/main.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"cloud-foundation-toolkit/config-connector/tests/ccs-test/cmd"
+	"github.com/golang/glog"
+)
+
+func main() {
```

Comment:
To verify, is CLI returning correct exit code (0 vs 1) for successful vs error execution?

Reply:
Yes, when there is something wrong in the CLI, it is terminated with log.Fatal, which exits with code 1. Otherwise, it is successful with exit code 0.